pkgs/applications/blockchains/btcpayserver/default.nix

```diff
@@ -34,8 +36,8 @@ stdenv.mkDerivation rec {
     nuget sources Add -Name tmpsrc -Source $TMP/nuget
     nuget init ${linkFarmFromDrvs "deps" deps} $TMP/nuget
 
-    dotnet restore --source $TMP/nuget BTCPayServer/BTCPayServer.csproj
-    dotnet publish --no-restore --output $out/share/$pname -c Release BTCPayServer/BTCPayServer.csproj
+    dotnet restore --source $TMP/nuget ${optionalString altcoinSupport ''/p:Configuration="Altcoins-Release" /p:RazorCompileOnBuild=false''} BTCPayServer/BTCPayServer.csproj
```

Comment:
Why is there a need for `RazorCompileOnBuild=false`? Can you add comment into the source code why is that needed?

Reply:
Actually not needed